Sources/VaporSecurityHeaders/Configurations/ReferrerPolicyConfiguration.swift

```diff
@@ -14,13 +14,17 @@ public struct ReferrerPolicyConfiguration: SecurityHeaderConfiguration {
         case unsafeUrl = "unsafe-url"
     }
 
-    private let option: Options
+    private let headerValue: String
 
-    public init(_ option: Options) {
-        self.option = option
+    public init(_ directive: Directive) {
+        headerValue = directive.rawValue
+    }
+
```

Comment:
This can just save the individual one
```suggestion
    public init(_ directive: Directive) {
        self.directives = [directive]
    }

```